indra/newview/skins/default/xui/ja/fonts.xml

```diff
@@ -2,11 +2,11 @@
 <fonts>
 	<font name="default" comment="default font files (global fallbacks)">
 		<file>
-			NotoSansCJKjp-Medium.otf
-		</file>
-		<file>
-			DejaVuSans.ttf
+			NotoSansCJKjp-SemiBold.otf
```

Comment:
Needs testing, if file doesn't load, might need to include NotoSansCJKjp into google fonts pacakge here https://github.com/secondlife/3p-google-fonts.